packages/graphiql-explorer/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "graphiql-explorer",
+  "version": "0.9.1",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "license": "MIT",
```

Comment:
some other thoughts about the package:

-  for GraphQL Foundation to adopt it, as when we adoped the FB (now meta) projects, we would need to convert it to MIT license
- you would still be attributed as the original authors
- we can include flow types for legacy flow typed compatibility if needed
- we can also of course attribute you as the original creators in the readme
- we can include a CJS export as well for backwards compatibility
- we can make this 0.10.0 to mark the breaking changes in type definition
- we can import and triage existing issues and PRs